plugin/src/main/java/io/jenkins/plugins/forensics/git/miner/GitRepositoryMiner.java

```diff
@@ -106,7 +114,7 @@ public RepositoryStatistics invoke(final Repository repository, final VirtualCha
 
         RepositoryStatistics analyze(final Repository repository, final Git git, final List<RevCommit> commits)
                 throws IOException {
-            RepositoryStatistics statistics = new RepositoryStatistics();
+            RepositoryStatistics statistics = new RepositoryStatistics(repository.resolve(Constants.HEAD).getName());
```

Comment:
I think `resolve` can return `null`, so in this case we should return empty statistics?